oak_remote_attestation_verification/src/verifier.rs

```diff
@@ -83,6 +89,7 @@ pub fn verify_binary_endorsement(
 
 /// Verifies endorsement against the given reference values.
 pub fn verify_endorsement_statement(
+    now_utc_millis: i64,
```

Comment:
Why not pass a more semantically accurate type like `std::time::Instant`? Is that because we need to keep this crate no_std and there is no equivalent in `core`? If so, maybe add a comment?

Reply:
(we can get back to adding the comment in the future, please feel free to submit as is)

Reply:
Yes, this may look like primitive obsession.

Reply:
Wait, on second thought, this crate doesn't need to be no_std , does it?